docs/Project.toml

```diff
@@ -1,3 +1,4 @@
 [deps]
 Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
+Revise = "295af30f-e4ad-537b-8983-00126c2a3abe"
```

Comment:
Don't think we need revise here. You can add it to the examples environment to help out development